src/server/builds/node.js

```diff
@@ -69,7 +69,8 @@ function algoliasearch(applicationID, apiKey, opts) {
 }
 
 algoliasearch.version = require('../../version.js');
-algoliasearch.ua = 'Algolia for Node.js (' + algoliasearch.version + ')';
+algoliasearch.ua = 'Algolia for Node.js (' + algoliasearch.version + '); ' +
```

Comment:
```suggestion
algoliasearch.ua = [
  'Algolia for Node.js (' + algoliasearch.version + ')',
  'Node.js (' + process.versions.node + ')'
].join('; ');
```

This would actually be more readable and less error-prone.